packages/interfaces/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Modification of `invoke` (`IAppliance`) to consume a PayloadArray.
+- Modification of `invoke` (`IAppliance`) to return a PayloadArray.
```

Comment:
…I'm now thinking you're right, we should at least tag here that this is a major change, so we can determine the right version number at publishing time, with the assumption that during our publishing we'll consult and remove those tags.

There's probably a formal way of doing this and yes the answer is probably conventional commits.

Reply:
I appreciate the trouble of conventional commits -- I also wish there was a conventional commit option that let you just use some kind of tag or note in the body of the commit message rather than the title.

On the one hand, we could just adopt that kind of rule ourselves, we are not bound to use conventional commits. What we lose is the potential for tooling that automatically updates versions -- that said, I'm not sure how conventional commits work in the context of monorepos with multiple versions.  Is lerna smart enough to know which commit applies to which repo?

I'm gonna spend a few moments looking to see what options exist and then pick a light weight approach for this one (probably just temporarily noting in change log for now).